test/test_pebble.py

```diff
@@ -935,6 +970,87 @@ def test_get_plan(self):
             ('GET', '/v1/plan', {'format': 'yaml'}, None),
         ])
 
+    def test_get_services_all(self):
+        self.client.responses.append({
+            "result": [
+                {
+                    "current": "inactive",
+                    "name": "svc1",
+                    "startup": "disabled"
+                },
+                {
+                    "current": "active",
+                    "name": "svc2",
+                    "startup": "enabled"
+                }
+            ],
+            "status": "OK",
+            "status-code": 200,
+            "type": "sync"
+        })
+        services = self.client.get_services()
+        self.assertEqual(len(services), 2)
+        self.assertEqual(services[0].name, 'svc1')
+        self.assertEqual(services[0].startup, pebble.ServiceStartup.DISABLED)
+        self.assertEqual(services[0].current, pebble.ServiceStatus.INACTIVE)
+        self.assertEqual(services[1].name, 'svc2')
+        self.assertEqual(services[1].startup, pebble.ServiceStartup.ENABLED)
+        self.assertEqual(services[1].current, pebble.ServiceStatus.ACTIVE)
+
+        self.assertEqual(self.client.requests, [
+            ('GET', '/v1/services', {'names': ''}, None),
```

Comment:
I would have expected the client to not send names if they were not specified.

Reply:
Yeah, true, that's a bit better -- fixed.